tools/wpt/browser.py

```diff
@@ -56,7 +57,7 @@ def __init__(self, logger):
     def _get_dest(self, dest, channel):
         if dest is None:
             # os.getcwd() doesn't include the venv path
-            dest = os.path.join(os.getcwd(), "_venv")
+            dest = os.path.join(os.getcwd(), venv_dir())
```

Comment:
This code looks weird. `dest` is set again right after this, so what's the point?

Reply:
This statement assumes that if we're not specifying a destination, the destination directory is the virtual environment directory.
https://github.com/web-platform-tests/wpt/blob/8a572e49654db1645351f87f5b242e75a23ab2e8/tools/wpt/browser.py#L58-L62
The last line here just appends the last part of the browser binary path to the destination i.e. `{dest}/browsers/{channel}`

I think maybe this method should have a better name and commenting, as it is a bit vague what exactly this should be used for.

Reply:
Oh, `dest` is used on like 62, I missed that!